front/src/routes/scene/edit-scene/index.js

```diff
@@ -270,6 +287,7 @@ class EditScene extends Component {
 
   componentDidMount() {
     this.getSceneBySelector();
+    this.getHouses();
```

Comment:
why do you get the house here? it means the house will be get no matter what, each time the page is loaded, even if this trigger is not used.

I don't think we want that, it should be in the SunriseSunsetTrigger.jsx component.